seemabansal2003.md

```diff
@@ -0,0 +1,55 @@
+---
+name: "@seemabansal2003"
+project: "TechnoFlora"
+---
+
+# TechnoFlora
+
+## Summary
+
+I come from a family of gardening enthusiasts who love to spend time with their plants. However whenever we go on a vacation, most of our plants die due to lack of care. Thus, being an Arduino Enthusiast, i want to build an automatic hydroponics based plant incubator called TechnoFlora. TechnoFlora would feature technologies like hydroponics, grow lights & temperature and humidity control that would not only help the plant in growing during adverse conditions but also faster than normal during better times. I can't wait to grow and completely harvest lettuce within just a couple of weeks!
+
+## Plan
+
+First of all, TechnoFlora would feature a hydroponics tray with water level indicators that can act as the medium of plant growth, nourishing the plant with the required minerals. Then it would also feature a full spectrum grow light which can artificially stimulate photosynthesis and help plants grow faster. Along with this, It would be covered with an acrylic enclosure that would not only trap stray light but also help in making a controlled environnment around the plant. There would be a temperature & humidity control system featuring a DHT11 sensor & vntilation and cooling fans to control the plants surroundings. All information regarding the plant's growth would be synced with a mobile application using bluetooth. This would not only help me monitor the plant better but also help it grow faster.
+
+## Budget
+
+
+| Product                                                | Supplier/Link                     | Cost   |
+| --------------------------------------------------------------------- | ---------------------------------------------- | ---------- |
+|hydroponics tray |https://amzn.eu/d/4M5MgpP| $43.50 |
+|Hydroponics Nutrient |https://amzn.eu/d/f0NNvWq| $5.14 |
+| TDS Meter |https://amzn.eu/d/dWoPVoC| $3.61 |
+| Grow Lights |https://amzn.eu/d/eA8CtlQ| $9.70 |
+|Blueooth Module |https://amzn.eu/d/3BkkD9c| $4.65 |
+| pH regulator solution |https://amzn.eu/d/4ktwq4j| $3.93 |
+| Temperature Sensing probe |https://amzn.eu/d/1A9TqwF| $1.92 |
+Peristaltic Pump |https://amzn.eu/d/8QuNLsl| $9.06 |
+| 12V Li-ion 8Ah battery |https://amzn.eu/d/07Z58gw| $26.31 |
+| 12 v battery charger |https://amzn.eu/d/9x9BDnY| $3.37 |
+| MDF 24 inch sq x3 |https://amzn.eu/d/b5YqREa| $6.40 |
+|Rotary tool for cutting, sanding & grinding |https://amzn.eu/d/7nctiV8| $30.20 |
```

Comment:
which part of the project is this for? acrylic cover?

Reply:
the rotary tool would be needed for a lot of things including making the cover & modifying the hydroponics tray to house the electronics.

Reply:
sounds good, thanks!